cpp/tests/CMakeLists.txt

```diff
@@ -598,6 +598,13 @@ set(MERGE_TEST_SRC
 
 ConfigureTest(MERGE_TEST "${MERGE_TEST_SRC}")
 
+###################################################################################################
+# - sorted-merge string tests ----------------------------------------------------------------------------
+
+set(MERGE_STRING_TEST_SRC
+  "${CMAKE_CURRENT_SOURCE_DIR}/merge/merge-str-test.cpp")
```

Comment:
Also please follow file naming conventions, and need to abbreviate. Should be `merge_string_test.cpp`

Reply:
Done.